### DISCUSSION TITLE
Dependency submission: The snapshot was accepted, but it is superseded by a newer snapshot from the same detector and correlator.

### DISCUSSION THREAD

**Discussion by andreostermeier27** [community/community#78276] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hello,

we want to ingest dependency information to our Scala based repositories using the dependency submission API as documented here: https://docs.github.com/en/rest/dependency-graph/dependency-submission?apiVersion=2022-11-28

I was just modifying the data and tried to ingest data into a test repository. However I am getting this as the API response:

`{
    "id": 9605706,
    "created_at": "2023-12-08T12:50:08.678Z",
    "result": "ACCEPTED",
    "message": "The snapshot was accepted, but it is superseded by a newer snapshot from the same detector and correlator. It will not update dependency results for the repository."
}`

I was googling for this error message but I did not find anything. Can someone help me understanding what I need to provide to make GitHub accepting my request?

Thanks

**Comment by EvgM2** [comment, marked as answer]
I've fixed this error by providing real commit sha from the "git log" command.
